README.md

```diff
@@ -184,4 +184,4 @@ Any user of this software shall indemnify and hold harmless Comma.ai, Inc. and i
 
 **THIS IS ALPHA QUALITY SOFTWARE FOR RESEARCH PURPOSES ONLY. THIS IS NOT A PRODUCT.
 YOU ARE RESPONSIBLE FOR COMPLYING WITH LOCAL LAWS AND REGULATIONS.
-NO WARRANTY EXPRESSED OR IMPLIED.**
+NO WARRANTY EXPRESSED OR IMPLIED.**
```

Comment:
this is already added to the internal README

Reply:
I'm thinking maybe it was a line spacing issue or something? I just pasted the main README.md from the previous version...